src/App.tsx

```diff
@@ -15,33 +17,63 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <div className="App">
-    <button type="button">
-      Start
-    </button>
-
-    <button type="button">
-      Sort alphabetically
-    </button>
-
-    <button type="button">
-      Sort by length
-    </button>
-
-    <button type="button">
-      Reverse
-    </button>
-
-    <button type="button">
-      Reset
-    </button>
-
-    <ul className="Goods">
-      <li className="Goods__item">Dumplings</li>
-      <li className="Goods__item">Carrot</li>
-      <li className="Goods__item">Eggs</li>
-      <li className="Goods__item">...</li>
-    </ul>
-  </div>
-);
+export const App: React.FC = () => {
+  const [start, setStart] = useState(true);
+  const [sortBy, setSortBy] = useState('');
+  const [reverse, setReverse] = useState(false);
+
+  const arrayCopy = [...goodsFromServer];
+
+  const sortHandler = () => {
+    arrayCopy.sort((a, b) => {
+      switch (sortBy) {
+        case 'alphabet':
+          if (!reverse) {
+            return a.localeCompare(b);
+          }
+
+          return b.localeCompare(a);
+          break;
+        case 'length':
+          if (!reverse) {
+            return a.length - b.length;
+          }
+
+          return b.length - a.length;
+          break;
+        default:
+          return 0;
+          break;
```

Comment:
no need to use break after return, and better to use ternary instead of `if `statement